src/cycax/cycad/features.py

```diff
@@ -279,3 +341,26 @@ def __init__(self, side: str, x: float, y: float, z: float, diameter: float):
         self.type = "cut"
         Location.__init__(self, x, y, z, side)
         self.diameter = diameter
+
+
+class SphereAdd(Feature):
```

Comment:
Could we not reuse the Sphere and just pass in the `type`: `cut` or `add`. Or if it is a switch then have a bool `cut=True`.